### PR TITLE
MES-4660: implement all mod1 category selection

### PR DESCRIPTION
**Pull request by szabikr**
## Description

Selecting the right function for different A Mod1 subcategories

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [x] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/providers/fault-count/fault-count.ts [hunk 1]
```diff
@@ -83,7 +92,10 @@ export class FaultCountProvider {
       case TestCategory.DE:
       case TestCategory.D: return sumManoeuvreFaults(data, faultType);
       // TODO: To be implemented properly in MES-4420
```
Comment (TomBillingtonUK) [inline src/providers/fault-count/fault-count.ts]: Can this TODO be removed yet

**Review by rossfauldsbjss — APPROVED** [review]
looks good

File: src/providers/fault-count/fault-count.ts [hunk 2]
```diff
@@ -100,7 +112,10 @@ export class FaultCountProvider {
       case TestCategory.DE: return FaultCountDHelper.getVehicleChecksFaultCountCatDE(data);
       case TestCategory.D: return FaultCountDHelper.getVehicleChecksFaultCountCatD(data);
       // TODO: To be implemented properly in MES-4420
```
Comment (TomBillingtonUK) [inline src/providers/fault-count/fault-count.ts]: same here - the ticket says it's done

**Review by TomBillingtonUK — APPROVED** [review]
Approved - Couple of TODO's which can potentially be removed

File: src/providers/fault-count/fault-count.ts (hunk 1 above)
Comment (szabikr) [reply src/providers/fault-count/fault-count.ts]: removed them

**Review by MarkPatt89 — APPROVED** [review]

**Pull request merged**